HW0/P6/P6B.py

```diff
@@ -16,16 +16,26 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10.0 ** ii for ii in range(-6, 0)]
```

Comment:
you might want to have more data points. A way to do this without too much manual work or double for loops is to use 
np.logspace(-6, 0) (gives 50 points by default), or np.logspace(-6, 0, x) where x is the total number of points you want